debian/control

```diff
@@ -11,7 +11,8 @@ Build-Depends: debhelper (>= 8.0.0),
  bison,
  libglib2.0-dev,
  libvirt-dev,
```

Comment:
If you are already editing this file might as well remove `libvirt-dev`, it's no longer required to be present for all cases, it gets automatically detected at runtime if it is.